aws/cloudformation/cloud_formation_stack.yml.erb

```
@@ -29,9 +29,8 @@ Parameters:
     Description: Subdomain name for corporate web site ("Pegasus").
   MarketingDomainName:
     Type: String
-    # Todo: update to handle multiple envs once the app is in prod
-    Default: 'dev.marketing.dev-code.org'
-    Description: Subdomain name for marketing site.
+    Default: <%="code.marketing-sites.#{CDO.rack_env == 'production' ? 'code.org' : 'dev-code.org'}"%>
```

There are many ways to test whether the template is being rendered on the production system. I'm not sure whether there's a particular way to do this that's better than others, but it's probably worth being consistent with how we do this conditional logic elsewhere in this template.
```suggestion
    Default: <%="code.marketing-sites.#{rack_env?(:production) ? 'code.org' : 'dev-code.org'}"%>
```